kolibri/core/assets/src/state/actions.js

```diff
@@ -174,6 +171,15 @@ function handleApiError(store, errorObject) {
   handleError(store, JSON.stringify(errorObject, null, '\t'));
 }
 
+function handleCoachPageError(store, errorObject) {
```

Comment:
Could you move this to the coach plugin, since it probably won't be used in the rest of the app.

You could use `Array.prototype.includes` to simplify the condition to be

```diff
-authErrorCodes.indexOf(errorObject.status.code) < 0
+authErrorCodes.includes(errorObject.status.code)
```